near-fetch/src/ops.rs

```diff
@@ -519,7 +520,11 @@ impl AsyncTransactionStatus {
     pub async fn status(&self) -> Result<Poll<ExecutionFinalResult>> {
         let result = self
             .client
-            .tx_async_status(&self.sender_id, self.hash)
+            .tx_async_status(
+                &self.sender_id,
+                self.hash,
+                Some(TxExecutionStatus::Executed),
```

Comment:
Should this be `Executed` for wait_until here?

Reply:
It's quite unclear how the `wait_until` parameter interacts with `RpcTransactionStatusRequest`. Let me see if I can get someone from protocol to comment on this